contact-widgets.php

```diff
@@ -2,7 +2,7 @@
 /**
  * Plugin Name: Contact Widgets
  * Description: Beautifully display social media and contact information on your website with these simple widgets.
- * Version: 1.3.3
+ * Version: 1.3.4
```

Comment:
@jonathanbardo I think the issue was that the version was bumped in the `master` branch, but the `develop` branch versions didn't get bumped (that was the only difference between the branches) - so I just pulled what we had on master and started working off that so everything was in sync moving forward.